### PR TITLE
Adds method guide cards from discovery category

### PR DESCRIPTION
**Pull request by jduss4**
Changed link style from `<a>` to markdown `[]()` for consistency

Roughs in three pages to prevent broken links:
- comparative analysis
- personas
- interview checklist

A portion of #234

### CODE REVIEW

File: content/methods/discover/design-studio.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  parent: Discover
+  title: Design studio
+method:
+  title: Design studio
+  what: An illustration-based way to facilitate communication (and brainstorming) between a project team and stakeholders.
+  why: To create a shared understanding and appreciation of design problems confronting the project team.
+  timeRequired: 3–4 hours
+  category: discover
+  content: ./_includes/methods/markdown/design-studio.md
+  last_modified_at: example timestamp
```
Comment (echappen) [inline content/methods/discover/design-studio.md:19]: @jduss4 I think we can remove these `last_modified_at` keys. They don't appear to be used on the old site.

File: content/methods/interview-checklist.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Interview checklist
+permalink: /methods/interview-checklist/
+layout: layouts/page
+---
```
Comment (echappen) [inline content/methods/interview-checklist.md:5]: @jduss4 [blocking request] Let's add the content for this page while we're at it.

Comment (jduss4) [reply content/methods/interview-checklist.md:5]: Can do!   Some of these pages link to other pages which link to other pages (including other categories) which is why I had held off initially.  I'll see how it goes when I get into it.

Comment (echappen) [reply content/methods/interview-checklist.md:5]: After discussing, we'll handle this in another PR

**Review by echappen — APPROVED** [review]

**Pull request merged**